diff --git a/dlls/d2d1/Makefile.in b/dlls/d2d1/Makefile.in
index c8adeaf..95cee67 100644
--- a/dlls/d2d1/Makefile.in
+++ b/dlls/d2d1/Makefile.in
@@ -1,7 +1,7 @@
 MODULE    = d2d1.dll
 IMPORTLIB = d2d1
-IMPORTS   = d3d10_1 dxguid uuid gdi32 user32 advapi32 d3dcompiler
-DELAYIMPORTS = dwrite xmllite ole32
+IMPORTS   = d3d10_1 dxguid uuid gdi32 user32 advapi32 d3dcompiler dwrite xmllite ole32
+DELAYIMPORTS =
 
 SOURCES = \
 	bitmap.c \
