diff --git a/dlls/d2d1/effect.c b/dlls/d2d1/effect.c
index 7ea6673..673e040 100644
--- a/dlls/d2d1/effect.c
+++ b/dlls/d2d1/effect.c
@@ -1071,6 +1071,18 @@ L"<?xml version='1.0'?>                                                   \
     </Inputs>                                                             \
   </Effect>";
 
+static const WCHAR color_management_description[] =
+L"<?xml version='1.0'?>                                                          \
+  <Effect>                                                                       \
+    <Property name='DisplayName' type='string' value='Color Management'/>        \
+    <Property name='Author'      type='string' value='The Wine Project'/>        \
+    <Property name='Category'    type='string' value='Stub'/>                    \
+    <Property name='Description' type='string' value='Color Management'/>        \
+    <Inputs >                                                                    \
+      <Input name='Source'/>                                                     \
+    </Inputs>                                                                    \
+  </Effect>";
+
 void d2d_effects_init_builtins(struct d2d_factory *factory)
 {
     static const struct builtin_description
@@ -1086,6 +1098,7 @@ void d2d_effects_init_builtins(struct d2d_factory *factory)
         { &CLSID_D2D1Crop, crop_description },
         { &CLSID_D2D1Shadow, shadow_description },
         { &CLSID_D2D1Grayscale, grayscale_description },
+        { &CLSID_D2D1ColorManagement, color_management_description },
     };
     unsigned int i;
     HRESULT hr;
